### PR TITLE
Update seed node info

### PR DESCRIPTION
**Pull request by ttl33**

### CODE REVIEW

File: pages/testnets/testnet_info.md [hunk 1]
```diff
@@ -33,13 +36,13 @@ The above info can be found in this [`v4-testnets` repository](https://github.co
   },
   "stakeCurrency": {
     "coinDenom": "DV4TNT",
-    "coinDecimals": 6,
+    "coinDecimals": 18,
     "coinMinimalDenom": "dv4tnt",
```
Comment (teddyding) [inline pages/testnets/testnet_info.md:40]: `coinMinimalDenom` should be `adv4tnt`

File: pages/testnets/testnet_info.md [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
+### Seed Node for `dydx-testnet-4`
 
+* Lavender.Five: `20e1000e88125698264454a884812746c2eb4807@seeds.lavenderfive.com:23856`
+* Crosnest: `87ee8de5f0f82af6ee6740a30f8844bbe6434413@seed.dydx-testnet.cros-nest.com:26656`
+* CryptoCrew: `38e5a5ec34c578dc323cbdd9b98330abb448d586@tenderseed.ccvalidators.com:29104`
+* StakingCabin: `182ab0015fb4b7d751b12a9c0162ac123445eac1@seed.dydx-testnet.stakingcabin.com:26656`
+* StakerSpace: `76b472b107ccf20c3d6c110c4a2a217306d2dedb@dydx-seed.staker.space:26656`
 
 ### Chain info
 ```json
```
Comment (teddyding) [inline pages/testnets/testnet_info.md:19]: Should we update `chainId` accordingly?

File: pages/testnets/testnet_info.md (hunk 1 above)
Comment (ttl33) [reply pages/testnets/testnet_info.md]: lol I left a comment before I sent out the PR but never submitted my comment. I was gonna ask you this

updated!

File: pages/testnets/testnet_info.md (hunk 2 above)
Comment (ttl33) [reply pages/testnets/testnet_info.md:19]: done!

File: pages/testnets/testnet_info.md [hunk 3]
```diff
@@ -51,8 +55,8 @@ The above info can be found in this [`v4-testnets` repository](https://github.co
   "feeCurrencies": [
     {
       "coinDenom": "DV4TNT",
-      "coinDecimals": 6,
-      "coinMinimalDenom": "dv4tnt",
+      "coinDecimals": 18,
+      "coinMinimalDenom": "adv4tnt",
```
Comment (teddyding) [inline pages/testnets/testnet_info.md:59]: Not 100% what this is used for but from the name, probably should add USDC here?

**Review by teddyding — APPROVED** [review]

**Pull request merged**